tests/validation/tests/v1_api/test_monitoring_v2.py

```diff
@@ -0,0 +1,94 @@
+from .common import *  # NOQA
+import pytest
+
+namespace = {'client': None, 'cluster': None, 'project': None}
+monitoring_v2_version = os.environ.get('RANCHER_MONITORING_V2_VERSION', '9.4.202')
```

Comment:
@sowmyav27 brought up a good point here at the Demo session. Do we want to retrieve these versions dynamically ?

Reply:
A new method is added for getting the latest chart version.